easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -109,7 +111,12 @@ def configure_step(self, srcdir=None, builddir=None):
 
         options_string = ' '.join(options)
 
-        command = ' '.join([self.cfg['preconfigopts'], 'cmake', options_string, self.cfg['configopts'], srcdir])
+        command = ' '.join([
+            self.cfg['preconfigopts'],
+            self.cfg['configure_cmd'],
```

Comment:
@mboisson We should be a bit more careful here, and use `self.cfg.get`, see https://github.com/ComputeCanada/easybuild-easyblocks/pull/2